src/stats/LatencyStatsPlugin.java

```diff
@@ -0,0 +1,81 @@
+// This file is part of OpenTSDB.
+// Copyright (C) 2015  The OpenTSDB Authors.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 2.1 of the License, or (at your
+// option) any later version.  This program is distributed in the hope that it
+// will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+// of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser
+// General Public License for more details.  You should have received a copy
+// of the GNU Lesser General Public License along with this program.  If not,
+// see <http://www.gnu.org/licenses/>.
+package net.opentsdb.stats;
+
+import com.stumbleupon.async.Deferred;
+import net.opentsdb.utils.Config;
+
+/**
+ * Used to calculate latency stats for a given operation. May work synchronously by collecting measurements and
+ * recording directly into a {@link StatsCollector} or may emit events to an external system which writes back
+ * via TSDB write channels (e.g. Telnet or HTTP).
+ */
```

Comment:
Does this mean it defo won't make 2.3?
On 24 Apr 2016 19:45, "Chris Larsen" <notifications@github.com> wrote:

> In src/stats/LatencyStatsPlugin.java
> <https://github.com/OpenTSDB/opentsdb/pull/769#discussion_r60849523>:
>
> > +// option) any later version.  This program is distributed in the hope that it
> > +// will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty
> > +// of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser
> > +// General Public License for more details.  You should have received a copy
> > +// of the GNU Lesser General Public License along with this program.  If not,
> > +// see <http://www.gnu.org/licenses/>.
> > +package net.opentsdb.stats;
> > +
> > +import com.stumbleupon.async.Deferred;
> > +import net.opentsdb.utils.Config;
> > +
> > +/**
> > + * Used to calculate latency stats for a given operation. May work synchronously by collecting measurements and
> > + * recording directly into a {@link StatsCollector} or may emit events to an external system which writes back
> > + * via TSDB write channels (e.g. Telnet or HTTP).
> > + */
>
> @since <https://github.com/since> 2.4
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenTSDB/opentsdb/pull/769/files/5252d0fbc0135b0376dc85774b60c1d409522508#r60849523>
>